eva-accession-import-dbsnp2/src/main/java/uk/ac/ebi/eva/accession/dbsnp2/model/DbsnpJson.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2019 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.accession.dbsnp2.model;
+
+import org.springframework.batch.core.configuration.annotation.StepScope;
+import uk.ac.ebi.eva.accession.core.ISubmittedVariant;
+
+import java.time.LocalDateTime;
+
+/**
+ * Core fields of a variant extracted from a dbSNP JSON file.
+ */
+@StepScope
+public class DbsnpJson implements ISubmittedVariant {
+
+    private String referenceSequenceAccession;
+
+    private int taxonomyAccession;
+
+    private String projectAccession;
+
+    private String contig;
+
+    private long start;
+
+    private String referenceAllele;
+
+    private String alternateAllele;
+
+    private Long clusteredVariantAccession;
+
+    public DbsnpJson() {
+        taxonomyAccession = 9606;
```

Comment:
I am a bit torn on doing this in the code. I understand that this code is primarily going to be for human import only but I was wondering if this is better supplied as an input parameter. @cyenyxe Any thoughts?

Reply:
Also, we should take in the assemblyAccession (GCA equivalent of the GCF assembly) as an input parameter 
![image](https://user-images.githubusercontent.com/25152678/60024818-65452c80-9690-11e9-85fc-8985a7b045af.png)

Reply:
suggestion: Derive from Input Param > hard-code

Reply:
> I am a bit torn on doing this in the code. I understand that this code is primarily going to be for human import only but I was wondering if this is better supplied as an input parameter. @cyenyxe Any thoughts?

I cringed a bit at first, but being honest this code is not "primarily going to be", it is "only going to be" for human imports for several years. Given that, I would make this a constant instead of a variable.

Reply:
please reopen if any follow-ups

Reply:
For assembly accession, my proposal would be to take the following two as input parameters:

1) Equivalent GCA accession for the GCF (see screenshot in the above comment). This will map to the referenceSequenceAccession member.
2) The GCF accession itself 

For a given pipeline run with the 2 given parameters above, the code should only use data in the "alleles" section under those documents where **primary_snapshot_data->placements_with_allele->placement_annot->seq_id_traits_by_assembly->assembly_accession** equals the GCF accession from the input parameter. @cyenyxe What do you think?